azure_monitor/examples/traces/trace.py

```diff
@@ -1,15 +1,17 @@
-from azure_monitor import AzureMonitorSpanExporter
 from opentelemetry import trace
 from opentelemetry.sdk.trace import TracerProvider
 from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
 
+from azure_monitor import AzureMonitorSpanExporter
+
 # Callback function to add os_type: linux to span properties
 def callback_function(envelope):
-    envelope.data.baseData.properties['os_type'] = 'linux'
+    envelope.data.baseData.properties["os_type"] = "linux"
     return True
 
+
 exporter = AzureMonitorSpanExporter(
-    connection_string='InstrumentationKey=99c42f65-1656-4c41-afde-bd86b709a4a7'
+    connection_string="InstrumentationKey=99c42f65-1656-4c41-afde-bd86b709a4a7"
```

Comment:
You should not have your actual ikey here

Reply:
Forgot to remove, thanks